docs/resources/Guild.md

```diff
@@ -544,3 +544,29 @@ Returns a partial [invite](#DOCS_RESOURCES_INVITE/invite-object) object for guil
 	"code": "abc"
 }
 ```
+
+## Get Guild Widget Image % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/widget.png
+
+Returns a PNG image widget for the guild. Requires no permissions or authentication.
+The same documentation also applies to `embed.png`.
+
+>info
+>All parameters for this endpoint are optional.
+
+###### Query String Params
+
+| Field | Type | Description |
+|-------|------|-------------|
+| style? * | string | style of the widget image returned (see below) |
+
+\* The default value for `style` if not set is `shield`.
```

Comment:
I believe this section should be made consistent with others that are similar:

- When specifying "all parameters for this endpoint are optional", use `field` instead of `field?`
- Add a `Default` column to show the default value

See example: [List Guild Members](https://discordapp.com/developers/docs/resources/guild#list-guild-members)

Reply:
Ty. Have made changes as such as I agree with this :)